EU4toV2/Source/V2World/Output/outCountry.cpp

```diff
@@ -117,29 +117,52 @@ std::ostream& V2::operator<<(std::ostream& output, const Country& country)
 
 void V2::Country::outputCommons(std::ostream& output)
 {
-	output << "graphical_culture = UsGC\n";	// default to US graphics
+	output << "graphical_culture = ";
+	auto graphicalCulture = modCommons.getGraphicalCulture();
+	if (graphicalCulture.empty())
+		graphicalCulture = details.graphicalCulture;
+	if (!graphicalCulture.empty())
+		output << graphicalCulture << '\n';
+	else
+		output << "UsGC\n"; // default to US graphics
+	
 	if (nationalColors.getMapColor())
 		output << "color " << *nationalColors.getMapColor() << "\n";
-	for (const auto& party : details.parties) output << party;
+	else if (!modCommons.getColor().empty())
+		output << "color " << modCommons.getColor() << '\n';
+	else if (!details.color.empty())
+		output << "color " << details.color << '\n';
+
+	auto parties = modCommons.getParties();
+	if (parties.empty())
```

Comment:
you can apply same logic here and below because it's cleaner. check modcommons, then check regular, then dump, no need to assign one over the other.